<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	border:3px solid red;
	border-radius:10px;
	background-color:pink;
	color:pink;
}
QToolButton{
	border-radius:10px;
	background-color:pink;
}
QToolButton:hover{
	border-radius:10px;
	font-size:17px;
	background-color:purple;
	color:yellow;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="Btn_Serch">
          <property name="text">
           <string>Serch</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTableView" name="tableView">
          <property name="minimumSize">
           <size>
            <width>171</width>
            <height>192</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="videowidget" name="m_widget" native="true">
          <property name="minimumSize">
           <size>
            <width>141</width>
            <height>192</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="minimumSize">
           <size>
            <width>111</width>
            <height>191</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lab_pixmap">
          <property name="minimumSize">
           <size>
            <width>141</width>
            <height>191</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QToolButton" name="tbtn_play">
          <property name="text">
           <string>开始</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbtn_pause">
          <property name="text">
           <string>暂停</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbtn_stop">
          <property name="text">
           <string>停止</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbtn_next">
          <property name="text">
           <string>下一首</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbtn_last">
          <property name="text">
           <string>上一首</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBox">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="specialValueText">
           <string/>
          </property>
          <property name="prefix">
           <string/>
          </property>
          <property name="suffix">
           <string>倍速</string>
          </property>
          <property name="decimals">
           <number>1</number>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_cycle">
          <property name="text">
           <string>循环播放</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="tbtn_mute">
          <property name="text">
           <string>声音</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="hSlider_sound">
          <property name="value">
           <number>30</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_name">
          <property name="text">
           <string>曲目</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="hSlider_progress">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_played">
          <property name="text">
           <string>已播放/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_time">
          <property name="text">
           <string>总时长</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuvideo">
    <property name="title">
     <string>video</string>
    </property>
    <addaction name="actionvideo"/>
    <addaction name="actionplainTextEdit"/>
    <addaction name="actionlabel"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件操作</string>
    </property>
    <addaction name="action_add"/>
    <addaction name="act_openfile"/>
    <addaction name="act_save"/>
    <addaction name="action_remove"/>
    <addaction name="action_clear"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menuvideo"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionvideo"/>
   <addaction name="actionplainTextEdit"/>
   <addaction name="actionlabel"/>
   <addaction name="separator"/>
   <addaction name="action_add"/>
   <addaction name="act_openfile"/>
   <addaction name="act_save"/>
   <addaction name="act_mysqlquery"/>
   <addaction name="act_mysqlinsert"/>
   <addaction name="action_remove"/>
   <addaction name="action_clear"/>
   <addaction name="separator"/>
   <addaction name="action_exit"/>
  </widget>
  <action name="actionvideo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示视频窗口</string>
   </property>
  </action>
  <action name="actionplainTextEdit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示文本窗口</string>
   </property>
  </action>
  <action name="actionlabel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示图片窗口</string>
   </property>
  </action>
  <action name="action_add">
   <property name="text">
    <string>添加</string>
   </property>
  </action>
  <action name="action_remove">
   <property name="text">
    <string>移出</string>
   </property>
  </action>
  <action name="action_clear">
   <property name="text">
    <string>清空</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="act_openfile">
   <property name="text">
    <string>打开文本</string>
   </property>
  </action>
  <action name="act_save">
   <property name="text">
    <string>保存文本</string>
   </property>
  </action>
  <action name="act_mysqlquery">
   <property name="text">
    <string>数据库导入</string>
   </property>
  </action>
  <action name="act_mysqlinsert">
   <property name="text">
    <string>导入数据库</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>videowidget</class>
   <extends>QWidget</extends>
   <header>videowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
